### DISCUSSION TITLE
Status 400

### DISCUSSION THREAD

**Discussion by githubUser01946** [victornpb/undiscord#498] [1. Questions & Answers]
`Error searching messages, API responded with status 400!
	{"code":50035,"errors":{"author_id":{"0":{"_errors":[{"code":"NUMBER_TYPE_COERCE","message":"Value \"@DiscordID#541414\" is not snowflake."}]}}},"message":"Invalid Form Body"}
CoreException	{}`

^ I get that.

How do I fix it please?

**Comment by Digital9999** [comment]
You should check all the filters, for some weird reason the 'before message' in interval message filter was auto filled for me with an e-mail of mine.

So just check all the filtering and make sure nothing is wrong.

**Reply by zachrdunn** [reply]
This is what the problem was thank you!

**Reply by kwjohnstone** [reply]
> You should check all the filters, for some weird reason the 'before message' in interval message filter was auto filled for me with an e-mail of mine.
> 
> So just check all the filtering and make sure nothing is wrong.

This was exactly the issue for me! Thanks dude! saved me so much time and head ache!
